plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -234,7 +234,10 @@ def process_main(self):
                 if content_type == man_list:
                     self._process_manifest_list(manifest, digest, available_blobs, tag)
                 else:
-                    self._process_manifest(manifest, digest, available_blobs, tag)
+                    foreign_layers = self._process_manifest(manifest, digest, available_blobs, tag)
+                    if foreign_layers:
+                        # we don't want to process schema1 manifest with foreign layers
```

Comment:
w/r to the blobs, yes we will get those which are not foreign type here https://github.com/pulp/pulp_docker/pull/201/files/57613e0fb5edc37b567653bb24429e2de00d6e3c#diff-cebb975e7cc2d51eddda339814fb7bd2R312